commitlint.config.cjs

```diff
@@ -1,3 +1,4 @@
-module.exports = {
-  extends: ["@commitlint/config-conventional"]
-};
+
+
+module.exports = { extends: ['@commitlint/config-conventional'] };
```

Comment:
Looks like this is just a linting change can you please revert it?
I don't think we need the single backtick change in here.